opm/common/OpmLog/LogUtil.cpp

```diff
@@ -52,6 +52,7 @@ namespace Log {
             prefix = "debug";
             break;
         case MessageType::Info:
+        case MessageType::Note:
```

Comment:
I think Note should have its own prefix (you must then also change the test). These are likely to be refined in the future though.